### ISSUE TITLE
More accurate kernel estimation, but worse SR result

### ISSUE THREAD

**Issue by greatlog** [yuanjunchai/IKC#11]
Hi, when I train IKC with a pertained SFTMD,  the loss of kernel maps will get smaller and smaller, which indicates that the estimation of kernel is getting more accurate, but the validation result on Set5 will first get better and then get worser. It seems that roughly accurate kernel estimation will  lead to better SR result that more accurate kernel estimation would.

I wonder if you come up with the same problem during training.

Thanks

**Comment by yuanjunchai** [comment]
Yes! I also found this phenomenon during training, and seemingly there is no certain connection between kernel accuracy and SR results. Here, my perspective is a more general kernel could lead to greater SR result than a more accurate kernel. Well, this is just my suggestion. Maybe you could find better way to explain this thing.
